search/includes/classes/class-queue.php

```diff
@@ -313,6 +313,10 @@ public function queue_object( $object_id, $indexable_slug = 'post', $options = a
 			return new WP_Error( 'invalid-indexable', sprintf( 'Indexable not found for type %s', $indexable_slug ) );
 		}
 
+		if ( ! \ElasticPress\Elasticsearch::factory()->index_exists( $indexable->get_index_name() ) ) {
```

Comment:
We can probably remove this check since we're already doing it in `queue_objects()` which calls `queue_object()`, yeah?